src/views/Help/SendFeedback/index.tsx

```diff
@@ -0,0 +1,469 @@
+): Promise<SendFeedbackEmailInput> {
+  const parsedValues: any = {};
+
+  // Map ui fields to backend fields
+  Object.entries(values).forEach(([field, value]) => {
+    // Handle easiServicesUsed separate from the rest since it is a list of options
+    if (field === 'easiServicesUsed') {
+      const easiServicesUsed = value as string[];
+
+      const easiServicesUsedOptionValue =
+        sendFeedbackOptions[
+          sendFeedbackOptionFieldsForTextInput.easiServicesUsed
+        ];
+
+      // Check the selected options list to see if the additional text is enabled
+      if (easiServicesUsed.includes(easiServicesUsedOptionValue)) {
+        parsedValues.easiServicesUsed = easiServicesUsed
+          .filter(optionValue => optionValue !== easiServicesUsedOptionValue)
+          .concat(
+            `${easiServicesUsedOptionValue}: ${values.easiServicesUsedAdditionalText}`
+          );
+      } else {
+        parsedValues.easiServicesUsed = easiServicesUsed;
+      }
+    }
+
+    // Parse additional text fields for option groups with one selection
+    else if (field in sendFeedbackOptionFieldsForTextInput) {
+      const additional: keyof SendFeedbackOptionFieldsWithTextInput = `${
+        field as SendFeedbackOptionFieldForTextInputKey
+      }${ADDITIONAL_TEXT_INPUT_SUFFIX}`;
+
+      const optionForTextInput =
+        sendFeedbackOptionFieldsForTextInput[
+          field as SendFeedbackOptionFieldForTextInputKey
+        ];
+
+      // Combine the original option value with additional text if
+      // the field group option for the additional text is selected
+      parsedValues[field] =
+        value ===
+        sendFeedbackOptions[optionForTextInput as SendFeedbackOptionKey]
+          ? `${value}: ${values[additional]}`
+          : value;
+    }
+
+    // Assign the remaining regular fields
+    else if (!field.endsWith(ADDITIONAL_TEXT_INPUT_SUFFIX)) {
+      parsedValues[field] = value;
+    }
+  });
+
+  const typedValues = sendFeedbackEmailInputSchema.cast(parsedValues);
+
+  // `canBeContacted` is false when `isAnonymous` is true
+  if (typedValues.isAnonymous) typedValues.canBeContacted = false;
+
+  // Make sure everything fits into the original backend type
+  const parsed = await sendFeedbackEmailInputSchema.validate(typedValues, {
+    strict: true
+  });
+
+  return parsed;
+}
+
+/**
+ * This formik form uses type `SendFeedbackEmailForm`, which is an extension
+ * of the original backend type `SendFeedbackEmailInput`.
+ */
+const SendFeedback = () => {
+  const { t } = useTranslation('help');
+
+  const [send] = useMutation<SendFeedbackEmail, SendFeedbackEmailVariables>(
+    SendFeedbackEmailQuery
+  );
+
+  const onSubmit = async (values: SendFeedbackEmailForm) => {
+    const input = await parseForm(values);
+    await send({ variables: { input } });
+  };
+
+  return (
+    <MainContent className="grid-container help-send-feedback">
+      <HelpBreadcrumb type={t('sendFeedback.closeTab')} />
```

Comment:
This translation equates to -> 'Close tab without sending feedback' when the conditional checks 'Close tab'.  

I don't think you should use a translation as a static prop.  Translation in nature is dynamic can can't be relied upon to always pass 'Close tab'.